### PR TITLE
[TextField] Update demo alert title to be variable.

### LINKED ISSUE #1611: [TextFields] Demo alert title doesn't change based on floating enabled/disabled.
### Overview
Using the catalog app viewing the Demo of TextFields, switching the style to use floating or not doesn't change the title of the alert which always says Floating Enabled (regardless or floating being enabled or not).

### Reproduction steps
Launch catalog and navigate to TextFields > Demo > STYLE. Notice in the alert title is Floating Enabled. Select No. Select STYLE to show alert again, the title still says Floating Enabled.

### Version number
 - 26.0.0

### Operating system and device
- iPhone 7 Plus - iOS 10.3 Simulator.

### PR DESCRIPTION
**Pull request by DonnaLea**
Update demo alert title to display based on whether the text fields are floating or not.

Closes #1611

### CODE REVIEW

**Comment by willlarche** [comment]
Thanks @DonnaLea for contributing! What was intended here was to explain what you are toggling which is "Whether floating is enabled" but it seems like that wasn't clear. Could you make this that the title is "Floating Labels" and there is a ✓ next to the one that is currently selected?

**Review by willlarche — CHANGES_REQUESTED** [review]
What was intended here was to explain what you are toggling which is "Whether floating is enabled" but it seems like that wasn't clear. Could you make this that the title is "Floating Labels" and there is a ✓ next to the one that is currently selected?

**Comment by DonnaLea** [comment]
Yep, that makes more sense! And yes, the problem was that it wasn't clear what the current state that was being represented was. Thanks for the suggestion, update coming.

**Comment by willlarche** [comment]
Thank you!!

**Comment by DonnaLea** [comment]
Updated with an amended commit. Ready for review. (can't see how to re-assign back)

File: components/TextFields/examples/TextFieldExample.swift [hunk 1]
```diff
@@ -214,16 +214,20 @@ final class TextFieldSwiftExample: UIViewController {
   }
 
   func buttonDidTouch(sender: Any) {
+    let isFloatingEnabled = allTextFieldControllers.first?.isFloatingEnabled ?? false
     let alert = UIAlertController(title: "Floating Enabled",
```
Comment (willlarche) [inline components/TextFields/examples/TextFieldExample.swift]: Let's make the title "Enable Placeholder Floating" and I'd say it's done!

**Review by willlarche — CHANGES_REQUESTED** [review]
Little change to title and it's ready.

File: components/TextFields/examples/TextFieldExample.swift (hunk 1 above)
Comment (willlarche) [reply components/TextFields/examples/TextFieldExample.swift]: Or "Floating Labels". Whichever you think is clearer.

**Review by willlarche — APPROVED** [review]

**Comment by DonnaLea** [comment]
My bad, that got lost in my edits. Fixed!

**Comment by willlarche** [comment]
Awaiting Travis CI, approval (which is a formality. There is no work in the framework.)

**Comment by willlarche** [comment]
Between when this was started and today, conflicts emerged. Correct them at your convenience and we'll try submitting again.

**Comment by ajsecord** [comment]
Will, GitHub claims this is ok to merge. Merge now?

**Pull request merged**